lib/recurly/subscription.php

```diff
@@ -4,7 +4,6 @@ class Recurly_Subscription extends Recurly_Resource
 {
   public function __construct($href = null, $client = null) {
     parent::__construct($href, $client);
-    $this->subscription_add_ons = array();
```

Comment:
Right, it works for purchases if we remove this line here, but breaks the library in other places.